packages/flutter/test/widgets/container_test.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/material.dart';
```

Comment:
uber nit: this looks a little weird in `widgets`, I guess it's so you can use `Colors`?  I have a vague hope that someday we could run tests based on what they import, and things like this will probably have to be rewritten. But it happens elsewhere, it's ok to leave for now.